Makefile

```diff
@@ -136,7 +136,9 @@ GDB ?= gdb-multiarch
 OUT_DIR ?= $(APP)
 
 APP_NAME := $(shell basename $(APP))
+ifndef LD_SCRIPT
```

Comment:
Add this argument to the documentation and initialization at the top of the file.